ods_ci/tests/Tests/0200__rhoai_upgrade/0201__pre_upgrade.robot

```diff
@@ -88,32 +79,72 @@ Verify User Can Disable The Runtime
+    [Setup]                 Begin Web Test
+    ${PRJ_TITLE}=           Set Variable            model-serving-upgrade
+    ${PRJ_DESCRIPTION}=     Set Variable            project used for model serving tests
+    ${MODEL_NAME}=          Set Variable            test-model
+    ${MODEL_CREATED}=       Set Variable            ${FALSE}
+    ${RUNTIME_NAME}=        Set Variable            Model Serving Test
+    ${INFERENCE_INPUT_OPENVINO}=    Set Variable
+    ...    @tests/Resources/Files/openvino-example-input.json
+    ${EXPECTED_INFERENCE_OUTPUT_OPENVINO}=    Set Variable
+    ...    {"model_name":"test-model__isvc-8655dc7979","model_version":"1","outputs":[{"name":"Func/StatefulPartitionedCall/output/_13:0","datatype":"FP32","shape":[1,1],"data":[0.99999994]}]}   # robocop: disable:line-too-long
+    ${runtime_pod_name}=    Replace String Using Regexp
+    ...    string=${RUNTIME_NAME}
+    ...    pattern=\\s
+    ...    replace_with=-
+    ${runtime_pod_name}=    Convert To Lower Case                           ${runtime_pod_name}
     Fetch CA Certificate If RHODS Is Self-Managed
     Clean All Models Of Current User
     Open Data Science Projects Home Page
-    Wait For RHODS Dashboard To Load    wait_for_cards=${FALSE}    expected_page=Data Science Projects
-    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
-    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
-    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
-    ...            aws_bucket_name=ods-ci-s3
-    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}
-    Serve Model    project_name=${PRJ_TITLE}    model_name=${MODEL_NAME}    framework=openvino_ir    existing_data_connection=${TRUE}
-    ...    data_connection_name=model-serving-connection    model_path=openvino-example-model
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds
-    ...  5 min  10 sec  Verify Openvino Deployment    runtime_name=${runtime_pod_name}
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Serving Service
-    Verify Model Status    ${MODEL_NAME}    success
-    Set Suite Variable    ${MODEL_CREATED}    ${TRUE}
-    Run Keyword And Continue On Failure    Verify Model Inference    ${MODEL_NAME}    ${INFERENCE_INPUT_OPENVINO}    ${EXPECTED_INFERENCE_OUTPUT_OPENVINO}    token_auth=${FALSE}
-    Remove File    openshift_ca.crt
-    [Teardown]   Run Keywords    Dashboard Test Teardown
-    ...    AND
-    ...    Run Keyword If Test Failed    Get Events And Pod Logs    namespace=${PRJ_TITLE}
-    ...    label_selector=name=modelmesh-serving-${runtime_pod_name}
+    Wait For RHODS Dashboard To Load
+    ...    wait_for_cards=${FALSE}
+    ...    expected_page=Data Science Projects
+    Create Data Science Project     title=${PRJ_TITLE}      description=${PRJ_DESCRIPTION}
```

Comment:
@apodhrad as this is backport and we do not have that approach in newer branches i think we should not do it in this pr (because if we will need to backport more changes than this will cause conflicts).